assets/styles/components/regulation/table.scss

```
@@ -9,13 +9,14 @@
             width: 10%;
         }
     }
-    .app-regulation-table__actions {
-        width: 25%;
-        @media screen and (min-width: 36em) {
-            width: 15%;
-        }
-        @media screen and (min-width: 48em) {
-            width: 20%;
-        }
+}
+
+.table__actions {
```

C'est vrai que ce nesting est complètement inutile et j'ai compris que l'idée était de pouvoir réutiliser cette classe

Par contre on devrait garder au moins le préfixe `app-`

```suggestion
.app-table__actions {
```